sympy/simplify/radsimp.py

```diff
@@ -1067,16 +1067,50 @@ def split_surds(expr):
 
 def _split_gcd(*a):
     """
-    split the list of integers ``a`` into a list of integers, ``a1`` having
-    ``g = gcd(a1)``, and a list ``a2`` whose elements are not divisible by
-    ``g``.  Returns ``g, a1, a2``
+    Parameters
+    =========
+
+    *a : integers, floats
+    atleast one integer or float should be passed as a parameter
+
+    Return
+    =========
+
+    (g, a1, a2) : a tuple with 3 elements
+
+    First element of the tuple is an int or a float(depending on the parameter),
+    second element is a list ``a1`` and third element is a list ``a2``.
+    The first element is g the gcd of the elements of the list ''a1``, and the
+    list ``a2`` contains those elements which are not divisible by ``g``.
+    The first element of the list `a1`` is always the first element of the list ``a``,
```

Comment:
Apart from this line, I don't think the added docstring adds anything new. The earlier one was quite simplified.

We can extend the earlier docstring by adding this line to it.